highfidelity/hifi
# Case 20585: Add missing Web Overlay functionality to Web Entities

https://highfidelity.manuscript.com/f/cases/20585/Add-missing-Web-Overlay-functionality-to-Web-Entities

Test plan:
- Run [this](https://gist.githubusercontent.com/SamGondelman/d4122d63b2f800f6e3e3fa741904c2fb/raw/6b82390cf7f13f01b18e5ae86e6d6b3144818a07/WebTest.js).  You'll see this:
![image](https://user-images.githubusercontent.com/7650116/51201441-ecf06900-18b0-11e9-9741-90edfca4a958.png)
- Press "o" several times.  The screens should all get higher resolution:
![image](https://user-images.githubusercontent.com/7650116/51201495-13ae9f80-18b1-11e9-9d3d-f5372c5d9e85.png)
- Press "i" several times.  The resolution will go back down:
![image](https://user-images.githubusercontent.com/7650116/51201527-27f29c80-18b1-11e9-98ab-ab732d311ec7.png)
- Press "u".  All 10 pages will reload with a new URL:
![image](https://user-images.githubusercontent.com/7650116/51201589-4d7fa600-18b1-11e9-91ff-18cfc8bd7f4e.png)
- Press "u" again.  All 10 pages will now display a gif:
![image](https://user-images.githubusercontent.com/7650116/51201620-5e301c00-18b1-11e9-8eca-f865aab718f8.png)
- Press "y".  The gifs should all run smoother.  Press it again and they'll all go back to a lower framerate.
- Verify that the tablet continues to work.  In particular: make sure you can still create entities with Create, and switch between domains with Goto.
- Run [this](https://gist.githubusercontent.com/SamGondelman/84d3b36c3fd0a29e1e2af105df23e33f/raw/9508e77c3e86d7876d3d5ddb650db7445727833d/WebEntityEventBridge.js).  You'll see this:
![image](https://user-images.githubusercontent.com/7650116/51202558-dac3fa00-18b3-11e9-80e3-ff06fbb770c9.png)
- When you press the buttons, colored cubes will appear in front of you:
![image](https://user-images.githubusercontent.com/7650116/51202628-07781180-18b4-11e9-9bde-1a945c419e17.png)
